src/components/Accounts/Transactions/Transaction/style.scss

```diff
@@ -1,4 +1,4 @@
-div.rounded.circle.emoji {
+div.emoji {
```

Comment:
Unsure whether this should be solely to just a `div`? eg `span`, `p` etc. Can we make this just `.emoji`?